packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1930,6 +1966,61 @@ export class ContainerRuntime
 		this.skipSavedCompressorOps = pendingRuntimeState?.pendingIdCompressorState !== undefined;
 	}
 
+	private async loadAndInitializeRuntimeAttributor(context: IContainerContext): Promise<void> {
+		if (this.runtimeAttributorP === undefined) {
```

Comment:
given that loadAndInitializeRuntimeAttributor() is called only once, and it's private, do we need all this complexity around tracking more state (like runtimeAttributorP, runtimeAttributorModuleLoaded, etc.)?
Feels like it could be simply async streigh-line function that does all the initialization. And if something fails, either we make whole container runtime load fail, or we just ignore it (I think later is not possible, as it will break eventual consistency)